README.md

```diff
@@ -1,339 +1,108 @@
-- Minimum: `docker-compose up proxy nextcloud` (nextcloud mysql redis mailhog)
-
-### Running stable versions
-
-The docker-compose file provides individual containers for stable Nextcloud releases. In order to run those you will need a checkout of the stable version server branch to your workspace directory. Using [git worktree](https://blog.juliushaertl.de/index.php/2018/01/24/how-to-checkout-multiple-git-branches-at-the-same-time/) makes it easy to have different branches checked out in parallel in separate directories.
-
-Note that for performance reasons the server repository might have been cloned with --depth=1 by default. To get the full history it is highly recommended to run:
-
-	cd workspace/server
-	git fetch --unshallow
-
- This may take some time depending on your internet connection speed.
-
-```
-cd workspace/server
-git worktree add ../stable23 stable23
-cd ../stable23
-git submodule update --init
+docker compose up nextcloud proxy
 ```
 
-After adding the worktree you can start the stable container using `docker-compose up -d stable23`. You can then add stable23.local to your `/etc/hosts` file to access it.
-
-Git worktrees can also be used to have a checkout of an apps stable brach within the server stable directory.
-
-```
-cd workspace/server/apps-extra/text
-git worktree add ../../../stable23/apps-extra/text stable23
-```
-
-### Running into errors
-
-If your setup isn't working and you can not figure out the reason why, running
-`docker-compose down -v` will remove the relevant containers and volumes,
-allowing you to run `docker-compose up` again from a clean slate.
-
-## 🔒 Reverse Proxy
-
-Used for SSL termination. To setup SSL support provide a proper DOMAIN_SUFFIX environment variable and put the certificates to ./data/ssl/ named by the domain name.
-
-You might need to add the domains to your `/etc/hosts` file:
+Ok, let's go to understand these commands line !
 
-```
-127.0.0.1 nextcloud.local
-127.0.0.1 collabora.local
-```
+First, you download the project with the git command, then you move to the `nextcloud-docker-dev` folder.
 
-This is assuming you have set `DOMAIN_SUFFIX=.local`
+The `bootstrap.sh` script check if all requirements are present and prepares your workspace.
+You have the `./workspace` folder where there is the `server` folder, the Nextcloud's core and [other Nextcloud versions](docs/running-stable-versions.md) if you want (the stable21, stable22, stable23, and so on).
 
-You can generate it through:
+So, you if you want to contribute to the Nextcloud's core, you can work in this folder directly !
 
-```
-awk -v D=.local '/- [A-z0-9]+\${DOMAIN_SUFFIX}/ {sub("\\$\{DOMAIN_SUFFIX\}", D " 127.0.0.1", $2); print $2}' docker-compose.yml
-```
+Then, you add `nextcloud.local` to your hosts file.
 
-You can generate selfsigned certificates using:
+Finally, the `docker compose up nextcloud proxy` command line. This command line runs the nextcloud, proxy, redis and mailhog containers.
```

Comment:
Should we also give an example how to stop with Ctrl+C and that you can run it in the background with `docker compose up -d ...`?

Reply:
Yes, I am writing how to stop with Ctrl+C. But, I don't know to run the containers in the background with the `-d` flag.

Because, it was my first error to run your project. Because, you see all information to understand what happen and you can connect on the instance when you see all users created :

```bash
nextcloud-nextcloud-1  |    The user "alice" was created successfully
nextcloud-nextcloud-1  |    The user "user6" was created successfully
nextcloud-nextcloud-1  |    The user "user1" was created successfully
nextcloud-nextcloud-1  |    The user "nextcloud" was created successfully
nextcloud-nextcloud-1  |    The user "user3" was created successfully
nextcloud-nextcloud-1  |    The user "user5" was created successfully
nextcloud-nextcloud-1  |    The user "jane" was created successfully
nextcloud-nextcloud-1  |    The user "john" was created successfully
nextcloud-nextcloud-1  |    The user "bob" was created successfully
nextcloud-nextcloud-1  |    The user "user4" was created successfully
nextcloud-nextcloud-1  |    The user "user2" was created successfully
```

When I used your project the first time, I didn't understand why I could not log in with the admin account. Because I run `docker-compose up -d proxy nextcloud` and immediately after, I tried to log in. It didn't work because the admin account didn't create ^^'

Reply:
I created a commit to fix it : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/e3594fe5c40bd14fc8f007701ce2564567413415

What do you think ?